server/src/main/java/keywhiz/KeywhizConfig.java

```diff
@@ -246,6 +249,14 @@ public void setMaximumSecretSizeInBytesInclusive(Long maximumSecretSizeInBytesIn
     this.maximumSecretSizeInBytesInclusive = maximumSecretSizeInBytesInclusive;
   }
 
+  public boolean shouldValidateDatabase() {
+    return validateDatabase == null ? true : validateDatabase;
+  }
```

Comment:
Note that since we historically always performed database validation, we default to validation if the field is unspecified for backwards compatibility.